src/query/sql/src/planner/binder/binder.rs

```diff
@@ -827,15 +827,26 @@ impl<'a> Binder {
     pub(crate) fn check_allowed_scalar_expr_with_subquery(
         &self,
         scalar: &ScalarExpr,
+        is_copy_into_table: bool,
```

Comment:
We can add a new check function only for `COPY INTO`, no need add `is_copy_into_table: bool`